NuGet.config

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <packageSources>
+    <add key="AspNetCore" value="https://dotnet.myget.org/F/aspnetcore-ci-dev/api/v3/index.json" />
```

Comment:
Could you copy the NuGet.config from https://github.com/aspnet/Common/blob/master/NuGet.config?